Zara Antonia-Maria/L4/3.13/main.c

```diff
@@ -0,0 +1,25 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+int main()
+{
+    float v[1001],value;
+    int n,i;
+    printf("Input the number of elements in the sequence: ");
+    scanf("%d",&n);
+    printf("\nInput the elements: ");
+    for(i=1;i<=n;i++)
+    {
+        scanf("%f",&v[i]);
+    }
+    printf("\nInput the value that is to be searched: ");
+    scanf("%f",&value);
+    i=1;
+    while(v[i]<value)
+    {
+        i++;
+    }
+    if(v[i]==value)printf("The value %f has been found on position %d",value,i);
```

Comment:
Pretty please refrain yourself from writing things like this. 
First off, format your code (`Format use AStyle`) and then put the appropriate curly brackets.

Many thanks!